src/components/lyne-timetable-barrier-free/styles/lyne-timetable-barrier-free.component.scss

```diff
@@ -0,0 +1,36 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin, available for shared
+// and default mode
+@include host-component-properties;
+
+:host {
+  --barrier-free-icon-size: 24;
+  --barrier-free-icon-padding-inline-start: calc(var(--barrier-free-icon-size) + var(--spacing-fixed-1x));
+}
+
+.barrier-free {
+  display: flex;
+  position: relative;
+  @include text-xs--regular();
+  line-height: pxToRem(var(--barrier-free-icon-size));
```

Comment:
the "expected" line-height would here be 20.4px defined by `@include text-xs--regular()` but this would lead to a slight difference due to the icon height.

Reply:
Since the icon is absolutly positioned we would consider to give the icon a little negative margin-top like: 
`margin-top: -1.8px;` ... (24px-20.4px)/2 ... could be a bit tricky with all the Sass/Custom Property calculations.

Reply:
The other option would be to use flexbox (align items center) instead of absolute positioning but leads to other multiline wrapping.

Reply:
You're right. I have fixed the position of the icon.